### PR TITLE
Marathon 20150620

### PR DESCRIPTION
**Pull request by jimwise**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not already refused in [closed issues].
- [X] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/marathon.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'marathon' do
+  version '20150620'
+  sha256 'd7e1a4063f19d247b738b1b856e48c24ea37b2543306de7608d65cb03d8d4327'
+
+  # github.com was verified as official when first introduced to the cask
+  url 'https://github.com/Aleph-One-Marathon/alephone/releases/download/release-20150620/Marathon-20150620-Mac.dmg'
```
Comment (miccal) [inline Casks/marathon.rb]: Please change this to ` url "https://github.com/Aleph-One-Marathon/alephone/releases/download/release-#{version}/Marathon-#{version}-Mac.dmg"`.

Also, please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/marathon.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'marathon' do
+  version '20150620'
+  sha256 'd7e1a4063f19d247b738b1b856e48c24ea37b2543306de7608d65cb03d8d4327'
+
+  # github.com was verified as official when first introduced to the cask
```
Comment (miccal) [inline Casks/marathon.rb]: Please change this to `# github.com/Aleph-One-Marathon/alephone was verified as official when first introduced to the cask`.

File: Casks/marathon.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  url 'https://github.com/Aleph-One-Marathon/alephone/releases/download/release-20150620/Marathon-20150620-Mac.dmg'
+  name 'marathon'
```
Comment (miccal) [inline Casks/marathon.rb]: Please change this to `name 'Marathon'`.

**Review by miccal — DISMISSED** [review]

**Comment by miccal** [comment]
Please make the same changes to your other two PR's.

Also, please remove the `marathon2.rb` file from this PR, thanks.

**Comment by jimwise** [comment]
Updated

File: Casks/marathon2.rb [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (vitorgalvao) [inline Casks/marathon2.rb]: You need to actually delete the file, not just empty it.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/marathon2.rb (hunk 4 above)
Comment (miccal) [inline Casks/marathon2.rb]: @vitorgalvao that we me, but I could not then delete it?

Comment (vitorgalvao) [inline Casks/marathon2.rb]: @miccal Can’t do it from the web interface, as far as I know.

Comment (vitorgalvao) [inline Casks/marathon2.rb]: @miccal Ah, you mean deleting after merging?

Comment (jimwise) [inline Casks/marathon2.rb]: removed

Comment (miccal) [inline Casks/marathon2.rb]: Thank you @jimwise.

Comment (jimwise) [inline Casks/marathon2.rb]: No, thank **you** @miccal and @vitorgalvao  -- gotta learn somehow.  👍

Comment (miccal) [inline Casks/marathon2.rb]: No problem @jimwise, but the file `marathon2.rb` is still there - are you not able to remove it?

**Comment by jimwise** [comment]
Try now?
-- 
				Jim Wise
				jwise@draga.com

(Message transmitted unsigned; digital signature available upon request)

**Review by vitorgalvao — APPROVED** [review]

**Comment by miccal** [comment]
Thanks @jimwise.

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**